server/models/availability.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require("mongoose");
+
+
+const dayInformation={
```

Comment:
Really like this 💯 No reason not to turn this into a schema of its own (hence converting it to a [subdocument](https://mongoosejs.com/docs/subdocs.html#subdocuments)) 👍🏼

Reply:
Thanks for sharing the document. It was very useful. Converted dayinformation into the dayInformationSchema.